dev/release/test-helper.rb

```diff
@@ -83,15 +83,60 @@ def parse_patch(patch)
       when /\A@@/
         in_hunk = true
         diffs.last[:hunks] << []
-      when /\A[-+]/
+      when /\A-/
         next unless in_hunk
         diffs.last[:hunks].last << line.chomp
+			when /\A\+/
+				next unless in_hunk
+				diffs.last[:hunks].last << normalize_added_line(line.chomp)
```

Comment:
I'll push indent fix for `dev/release/*.rb` changes.

Reply:
My bad (bad vim configuration). Let me fix the indent. Sorry about that.

Reply:
No problem. I've fixed the indent.

Reply:
I pushed the fix as well.

Reply:
Oh, why...?
You broke it again...: https://github.com/apache/arrow/compare/6d91fddd6ce3bfa28cbb25421cb671b1b737147e..cb22662007ecada122cd6d441546e3385c31e906#diff-7655978b5793694e21d21f4c483847ed0004cd1035ebd91a87c862c246bbb83a

You don't need to squash all your commits in a PR because we use "squash & merge".

Reply:
Ah ok. On most of other Apache projects I'm working we do a single commit to use simple merge. Sorry. I will fix indent.